Image_Generation_Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM nvidia/cuda:11.8.0-devel-ubuntu22.04
+LABEL name="unified-io-inference"
```

Comment:
Does it make sense to change the name to something more specific like `unified-io-inference-image-generation`?